module_bindings/scarab_param/scarab_param_namespace_pybind.cc

```diff
@@ -9,10 +9,16 @@
 #include "param_value_pybind.hh"
 #include "param_array_pybind.hh"
 #include "param_node_pybind.hh"
+#include "signal_handler.hh"
 
 PYBIND11_MODULE( scarab, scarab_mod )
 {
-    pybind11::module param_mod = scarab_mod.def_submodule( "param", "param data types" );
+    pybind11::module param_mod = scarab_mod.def_submodule( "param", "param data types" )
+	.def("cancel_all",
```

Comment:
I'm pretty iffy about implementing this directly in the source file, it really "should" get a header, it is small now, but that way there's a place to add anything else we later decide we need.

Either way this doesn't belong in the "param" submodule, it should go directly in the scarab module. (Actually, Noah pointed out that the param subdirectory in scarab doesn't have a separate library, so the param-related exporters should all be called directly on the scarab_module too, but that part isn't in your changes so that can be outside the scope of this PR.